OrbitLinuxTracing/TracerThread.cpp

```diff
@@ -828,6 +841,29 @@ void TracerThread::ProcessSampleEvent(const perf_event_header& header,
     thread_name.set_timestamp_ns(event->GetTimestamp());
     listener_->OnThreadName(std::move(thread_name));
 
+  } else if (is_user_instrumented_tracepoint) {
+    auto it = ids_to_tracepoint_info_.find(stream_id);
+
+    if (it != ids_to_tracepoint_info_.end()) {
+      auto event = ConsumeGenericTracepointPerfEvent(ring_buffer, header);
+
+      if (event->GetPid() != pid_) {
+        return;
+      }
+
+      orbit_grpc_protos::TracepointEvent tracepoint_event;
+      tracepoint_event.set_pid(event->GetPid());
+      tracepoint_event.set_tid(event->GetTid());
+      tracepoint_event.set_time(event->GetTimestamp());
+      tracepoint_event.set_cpu(event->GetCpu());
+
+      orbit_grpc_protos::TracepointInfo* tracepoint = tracepoint_event.mutable_tracepoint_info();
+      tracepoint->set_name(it->second.name());
+      tracepoint->set_category(it->second.category());
+
+      // TODO: Sending of the event
+      it++;
```

Comment:
`it++;` 

This does not make much sense. Is this a left over from a previous version? Please remove.

Reply:
Yes, it is from the previous version. I will delete it.